dvc/commands/add.py

```diff
@@ -51,6 +51,7 @@ def run(self):
                 to_remote=self.args.to_remote,
                 remote_jobs=self.args.remote_jobs,
                 force=self.args.force,
+                relink=self.args.relink,
```

Comment:
Based on https://github.com/iterative/dvc/pull/10389/files#r1565145737.

```suggestion
                relink=not self.args.no_relink,
```